polybar-scripts/player-spotify/player-spotify.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+
+main() {
+  if ! pgrep -x spotify >/dev/null; then
+    echo ""; exit
+  fi  
```

Comment:
Please use an if-else structure. Exiting somewhere in the code is a bit unclean.